tests/testthat/_snaps/expect-setequal.md

```diff
@@ -1,67 +1,73 @@
 # useful message on failure
 
-    "actual" (`actual`) and "expected" (`expected`) don't have the same values.
+    "actual" doesn't have the same values as "expected".
     * Only in `actual`: "actual"
     * Only in `expected`: "expected"
     
 
 ---
 
-    1:2 (`actual`) and 2 (`expected`) don't have the same values.
+    `x` doesn't have the same values as `y` (2).
     * Only in `actual`: 1
     
 
 ---
 
-    2 (`actual`) and 2:3 (`expected`) don't have the same values.
+    `x` (2) doesn't have the same values as `y`.
     * Only in `expected`: 3
     
 
 ---
 
-    1:2 (`actual`) and 2:3 (`expected`) don't have the same values.
+    `x` doesn't have the same values as `y`.
     * Only in `actual`: 1
     * Only in `expected`: 3
     
 
 ---
 
-    c("a", "a") (`actual`) and c("b", "b", "b") (`expected`) don't have the same values.
+    `x` doesn't have the same values as `y`.
     * Only in `actual`: "a"
     * Only in `expected`: "b"
     
 
+---
+
+    `x` doesn't have the same values as c("a", "b", "c", "d").
+    * Only in `expected`: "d"
+    
+
 # truncates long vectors
 
-    1:2 (`actual`) and 1:50 (`expected`) don't have the same values.
+    `x` doesn't have the same values as `y`.
     * Only in `expected`: 3, 4, 5, 6, 7, 8, 9, 10, 11, ...
     
 
 # expect_contains() gives useful message on failure
 
-    `x1` (`actual`) doesn't fully contain all the values in `x2` (`expected`).
-    * Missing from `actual`: "d"
-    * Present in `actual`:   "a", "b", "c"
```

Comment:
That's actually a case where the parameter names are useful and add context (as opposed to e.g. `object`).